manifests/check.pp

```diff
@@ -116,16 +116,34 @@
 
   $check_name = regsubst(regsubst($name, ' ', '_', 'G'), '[\(\)]', '', 'G')
 
-  file { "/etc/sensu/conf.d/checks/${check_name}.json":
+  case $::osfamily {
+    'windows': {
+      $etc_dir = 'C:/opt/sensu'
+      $conf_dir = "${etc_dir}/conf.d"
+      $user = undef
+      $group = undef
+      $file_mode = undef
```

Comment:
Not a windows user, but no defaults for these?  Does the sensu package installer create a user or does it just run as a system service?  Any way to keep other random users from browsing the sensu config which may contain sensitive information such as passwords?